happiness-frontend/src/components/HappinessCard.js

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import { CardActionArea } from "@mui/material";
+
+export default function HappinessCard({ data, click }) {
+  const date = new Date(data.timestamp + "T00:00:00");
+
+  return (
+    <Card sx={{ my: 1, boxShadow: 0, border: 1, borderColor: "#808080" }}>
+      <CardActionArea onClick={() => click()}>
```

Comment:
When passing a function in, instead of declaring an arrow function to call the function you can just directly pass the function in as reference, for example `onClick={click}`